examples/api-call/api_call.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+import asyncio
+import base64
+
+from yapapi import Golem
+from yapapi.services import Service
+from yapapi.log import enable_default_logger
+from yapapi.payload import vm
+
+OUTPUT_PATH = "/golem/work/response.txt"
+
+
+class ApiCallService(Service):
+    @staticmethod
+    async def get_payload():
+        manifest = open("manifest.json", "rb").read()
+        manifest = base64.b64encode(manifest).decode("utf-8")
+
+        manifest_sig = open("manifest.json.base64.sign.sha256", "rb").read()
+        manifest_sig = base64.b64encode(manifest_sig).decode("utf-8")
+
+        manifest_sig_algorithm = "sha256"
+
+        # both DER and PEM formats are supported
+        manifest_cert = open("requestor.cert.der", "rb").read()
+        manifest_cert = base64.b64encode(manifest_cert).decode("utf-8")
+
+        return await vm.manifest(
+            manifest=manifest,
+            manifest_sig=manifest_sig,
+            manifest_sig_algorithm=manifest_sig_algorithm,
+            manifest_cert=manifest_cert,
+            min_mem_gib=0.5,
+            min_cpu_threads=0.5,
+            capabilities=["inet", "manifest-support"],
+        )
+
+    async def start(self):
+        async for script in super().start():
+            yield script
+
+        script = self._ctx.new_script()
+        script.run(
+            "/bin/sh",
+            "-c",
+            f"GOLEM_PRICE=`curl -X 'GET' \
+                    'https://api.coingecko.com/api/v3/simple/price?ids=golem&vs_currencies=usd' \
+                    -H 'accept: application/json' | jq .golem.usd`; \
+                echo \"Golem price: $GOLEM_PRICE USD\" > {OUTPUT_PATH};",
+        )
```

Comment:
Moved it. It was my first golem app/service. I assumed Golem service requires both `start` and `run` because both `run_service` doc and hello-service examples have these methods implemented. With `run` only it looks like `execute_tasks` with single task.